kratos/processes/levelset_convection_process.h

```diff
@@ -517,10 +547,18 @@ class LevelSetConvectionProcess
         // Generating the elements
         mpDistanceModelPart->Elements().reserve(rBaseModelPart.NumberOfElements());
         for (auto it_elem = rBaseModelPart.ElementsBegin(); it_elem != rBaseModelPart.ElementsEnd(); ++it_elem){
-            Element::Pointer p_element = Kratos::make_intrusive< LevelSetConvectionElementSimplex < TDim, TDim+1 > >(
-                it_elem->Id(),
-                it_elem->pGetGeometry(),
-                it_elem->pGetProperties());
+            Element::Pointer p_element;
+            if (mIsAlgebraicStabilization){
+                p_element = Kratos::make_intrusive< LevelSetConvectionElementSimplexAlgebraicStabilization < TDim, TDim+1 > >(
+                    it_elem->Id(),
+                    it_elem->pGetGeometry(),
+                    it_elem->pGetProperties());
+            } else {
+                p_element = Kratos::make_intrusive< LevelSetConvectionElementSimplex < TDim, TDim+1 > >(
```

Comment:
I wonder if the element type could be a setting. It'd be great for us as it will allow us to use this process with different formulations...

Reply:
In the current implementation, the element type is set by "algebraic_stabilization" in the settings; **false**: LevelSetConvectionElementSimplex, and **true**: LevelSetConvectionElementSimplexAlgebraicStabilization.
Please let me know if you have any other suggestions.

Reply:
well I was thinking that maybe the type of element (if it algebraic_stabilization or any other formulation) could be set outside this process as a setting in a way that this process only receives the string of the element type. Im not completely sure if its possible though

Reply:
If the element is registered we can call the `Create` method of the prototype. Do you have plans to add different formulations (if there is only two options I think we can live with the current true/false value)?

Reply:
Yep.. the point is that we have some different formulations for this in our own repository (sadly we are not allowed to share them here) and that will allow us to use them using the same process..

Reply:
OK. We can encapsulate the element type selection in one function so you can just override this one to add your elements. This would do the job right?

Reply:
that would work yep , thanks !

Reply:
Just mentioning that we will address this in an upcoming PR.